docs/release.rst

```diff
@@ -0,0 +1,77 @@
+Release Process
+===============
+
+Location: :doc:`/index` → :doc:`/release`
+
+The two goals of the release process are to create a tagged commit which
+bumps the version file and describes changes since the last release, and to
+generate packages which can be uploaded to PyPI.
+
+There are other ways to create a *release* this one is just an example.
+
+Assumptions
+-----------
+
+This release process was created assuming that your **fork** is named **origin**,
+and the **upstream** is repository is named **upstream**.
+
+Steps
+-----
+
+1. Assume that your master branch is up to date.
+
+.. code-block:: sh
+
+    git checkout master
+    git fetch --all
+    git merge upstream/master
+    git status
+
+2. Clean any unnecessary files. Mainly the ones in **dist** dir.
+
+.. code-block:: sh
+
+    make dist-lean
+
+3. From the root of Pulp Smash dir verify the current version.
+
+.. code-block:: sh
+
+    cat VERSION 
+
+4. From the root of Pulp Smash dir run the script to create a new release.
+
+.. code-block:: sh
+
+    git status
+    ./scripts/release.sh 'string_new_version'
+
```

Comment:
@kersommoura worth mentioning that you should not have a virtualenv activated when running this script?

Reply:
Let me verify this again.

Reply:
There was no problem running the script with a python virtual env activated. 

https://github.com/PulpQE/pulp-smash/blob/1ee2d133a1080254a85fe0cf89b898fd8b9de897/scripts/release.sh#L75